languages/js/yarn.lock

```diff
@@ -2,29 +2,29 @@
 # yarn lockfile v1
```

Comment:
seems like the prevailing guidance is still to keep JS package manager lockfiles checked in, but man unnecessary ordering changes like this are annoying

Reply:
Oof, yeah. I didn't realise it was such a large change. I imagine this is from different yarn versions? Should we standardise on one?